packages/client/src/components/inspector/InspectorStateField.vue

```diff
@@ -16,10 +16,10 @@ const props = withDefaults(defineProps<{
 
 const state = useStateEditorContext()
 const bridgeRpc = useDevToolsBridgeRpc()
-const value = formatInspectorStateValue(props.data.value)
-const type = getInspectorStateValueType(props.data.value)
+const value = computed(() => formatInspectorStateValue(props.data.value))
+const type = computed(() => getInspectorStateValueType(props.data.value))
```

Comment:
In the default branch, these values will be updated only when the component is re-rendered.